distribution/ddf-common/src/main/resources/etc/org.apache.karaf.features.xml

```diff
@@ -65,6 +65,9 @@
         <bundle originalUri="mvn:org.apache.felix/org.apache.felix.scr/2.1.2"
                 replacement="mvn:org.apache.felix/org.apache.felix.scr/2.0.14"
                 mode="maven"/>
+        <bundle originalUri="mvn:org.apache.aries.proxy/org.apache.aries.proxy/1.1.2"
+                replacement="mvn:org.apache.aries.proxy/org.apache.aries.proxy/1.1.4"
+                mode="maven"/>
```

Comment:
Disclaimer: I worked with @Kjames5269 on this, so I'm biased.

The aries.proxy is only used one place in Karaf: https://github.com/apache/karaf/blob/5057aa441eb2df2f676d669b5bf0a5f223064e9f/assemblies/features/standard/src/main/feature/feature.xml#L33-L41. You can see this feature started in DDF:

```
admin@root()> feature:list -s -b | grep aries-proxy
aries-proxy                                     │ 4.2.1            │          │ Started     │ standard-4.2.1                         │ Aries Proxy                                        │ no
```

There are only 2 changes between those versions: https://github.com/apache/aries/commits/trunk/proxy/proxy-impl
https://github.com/apache/aries/commit/c1e12d86e09dbd86182cc32e7b4e458bb5c86dc1#diff-c08912c208952f4d6dae89c9cdb69b14 and https://github.com/apache/aries/commit/cceac80922bff4c6bd0680a92b4fe490d38c1659#diff-c08912c208952f4d6dae89c9cdb69b14, so it looks safe to bump.

We also look into upgrading Karaf to 4.2.3 (planned to be released next week), instead, which include the version bump fix, and decided that just upgrading the aries/proxy version is a safer, quicker option.

I am not sure about if this `bundleReplacement` is the right solution for upgrading the version that Karaf uses.

Reply:
We haven't talked about it much but I would like to avoid another karaf upgrade on the release branch if at all possible. This seems like a reasonable release branch solution.